### PR TITLE
fix: remove smoothscroll lib

### PR DESCRIPTION
**Pull request by duidae**
P.S. this PR will add corresponding updates of npm packages later.

### CODE REVIEW

File: src/components/about-us/index.js [hunk 1]
```diff
@@ -46,7 +45,10 @@ export class AboutUs extends PureComponent {
     this.sectionOffset = []
   }
   _handleClickAnchor = anchorIdx => {
-    return smoothScroll(this.sectionOffset[anchorIdx])
+    window.scroll({
+      top: this.sectionOffset[anchorIdx],
+      behavior: 'smooth',
```
Comment (taylrj) [inline src/components/about-us/index.js:50]: It seems to me that [Safari does not have support for the smooth scroll behavior](https://developer.mozilla.org/en-US/docs/Web/API/Window/scroll#browser_compatibility) ... 🥲

Comment (duidae) [reply src/components/about-us/index.js:50]: oh no...:cry

Comment (taylrj) [reply src/components/about-us/index.js:50]: https://github.com/iamdustan/smoothscroll

polyfill ?

Comment (duidae) [reply src/components/about-us/index.js]: replaced with our custom smooth scroll.

**Pull request merged**